Client/mods/deathmatch/logic/luadefs/CLuaVectorGraphicDefs.cpp

```diff
@@ -56,6 +56,22 @@ bool CLuaVectorGraphicDefs::LoadFromData(lua_State* luaVM, CClientVectorGraphic*
     return true;
 }
 
+bool CLuaVectorGraphicDefs::LoadFromNode(lua_State* luaVM, CClientVectorGraphic* pVectorGraphic, CXMLNode* pNode)
+{
+    if (!luaVM || !pVectorGraphic)
+        return false;
+
+    if (!pVectorGraphic->SetDocument(pNode))
+    {
+        delete pVectorGraphic;
+
+        m_pScriptDebugging->LogCustom(luaVM, "Unable to load SVG data (check for XML syntax errors)");
```

Comment:
![image](https://user-images.githubusercontent.com/12902714/150028482-4900908b-7c3e-4c11-a2d8-a183ab391586.png)


No, like honestly, you're calling this from a secondary thread, this is a bomb waiting to explode.